freqtrade/persistence/migrations.py

```diff
@@ -47,6 +47,14 @@ def migrate_trades_table(decl_base, inspector, engine, table_back_name: str, col
     min_rate = get_column_def(cols, 'min_rate', 'null')
     sell_reason = get_column_def(cols, 'sell_reason', 'null')
     strategy = get_column_def(cols, 'strategy', 'null')
+
+    leverage = get_column_def(cols, 'leverage', '0.0')
```

Comment:
In order for migrations to work correctly - you'll have to enter one of the new columns around row 177 (and for the orders table in row 191).

currently the bot crashes when started with an existing database as the migrations don't run - so the new tables will be missing.

Reply:
Could you explain what you mean a little more in depth here? Your message isn't ringing any bells for me.

Reply:
In this file - look at line 177 in this file (and 191 ... at least based on yesterday's count). 
It's pointing to the latest column that was added to the table - to detect if a migration is necessary or not.

if you don't update, freqtrade will not realize migrations need to run ... -> they will not run - and we'll end up having missing columns.